internal/store/bigtable/group.go

```diff
@@ -135,40 +117,3 @@ func (g *Group) UpdateBranchTable(branchTable *Table) {
 	g.tables = tables
 	SortTables(g.tables)
 }
-
-// NewTable creates a new cbt.Table instance.
-func NewBtTable(ctx context.Context, projectID, instanceID, tableID string) (
-	*cbt.Table, error) {
-	btClient, err := cbt.NewClient(ctx, projectID, instanceID)
-	if err != nil {
-		return nil, err
-	}
-	return btClient.Open(tableID), nil
-}
-
-// SortTables sorts the bigtable by import group preferences
-// - frequent should always be the highest rank
-// - infrequent should always be the lowest rank
-// - if a group is not in ranking list, put it right before "infrequent" and
-//   after other groups with ranking.
-func SortTables(tables []*Table) {
```

Comment:
imo, group felt like a better home than table for this?  (these are "group" operations)